src/Main.java

```diff
@@ -1,5 +1,92 @@
+import java.util.Formatter;
+
 public class Main {
+    private static Formatter Arrays;
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задача 1");
+
+        int[] weight = new int[]{1, 2, 3};
+        for (int i = 0; i < weight.length; i++) {
+            System.out.println(weight[i]);
+        }
+
+        double[] weight2 = new double[]{1.57, 7.654, 9.986};
+        for (int i = 0; i < weight2.length; i++) {
+            System.out.println(weight2[i]);
+        }
+
+        int[] weights = new int[11];
+        weights[0] = 984;
+        weights[9] = 6454;
+        for (int i = 0; i < weights.length; i++) {
+            System.out.println(weights[i]);
+        }
+
+        System.out.println("Задача 2");
+        weight = new int[]{1, 2, 3};
+        for (int i = 0; i < weight.length; i++) {
+            System.out.print(weight[i]);
+            if (i != 2) {
+                System.out.print(", ");
+            }
+        }
+        System.out.println(" ");
+        weight2 = new double[]{1.57, 7.654, 9.986};
+        for (int i = 0; i < weight2.length; i++) {
+            System.out.print(weight2[i]);
+            if (i != 2) {
+                System.out.print(", ");
+            }
+        }
+        System.out.println(" ");
+        weights = new int[11];
+        weights[0] = 984;
+        weights[9] = 6454;
+        for (int i = 0; i < weights.length; i++) {
+            System.out.print(weights[i]);
+            if (i != 10) {
```

Comment:
```suggestion
            if (i != weights.length - 1) {
```
используем длину массива, а не число с потолка